lib/resty/healthcheck.lua

```diff
@@ -366,8 +366,9 @@ end
 -- default is `true`.
 -- @param hostheader (optional) a value to use for the Host header on
 -- active healthchecks.
+-- @param tbl_meta (optional) a lua table with custom info of business stuff
 -- @return `true` on success, or `nil + error` on failure.
-function checker:add_target(ip, port, hostname, is_healthy, hostheader)
+function checker:add_target(ip, port, hostname, is_healthy, hostheader, tbl_meta)
```

Comment:
Can we use `meta` rather than `tbl_meta`?

Reply:
LGTM, `tbl_meta` means `table` type

Reply:
IMO, The "tbl_" prefix helps others to better understand what they are doing when looking at the code or making modifications.

Reply:
OK.